routers/repo/migrate.go

```diff
@@ -188,3 +193,53 @@ func MigratePost(ctx *context.Context, form auth.MigrateRepoForm) {
 
 	handleMigrateError(ctx, ctxUser, err, "MigratePost", tpl, &form)
 }
+
+// CancelMigration cancels a running migration
+func CancelMigration(ctx *context.Context) {
+	if !ctx.Repo.IsOwner() {
```

Comment:
```suggestion
	if !ctx.Repo.IsAdmin() {
```
I think that suffices.